src/app/main/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.ts

```diff
@@ -77,27 +83,61 @@ export class UBSAddAddressPopUpComponent implements OnInit, OnDestroy {
       id: [this.data.edit ? this.data.address.id : 0],
       actual: true
     });
+
+    // TODO: Must be removed if multi-city feature need to be implemented
+    this.onCitySelected('Kiev');
+  }
+
+  onCitySelected(citySelected) {
+    this.cities.forEach((city) => {
+      if (city.cityName === citySelected) {
+        this.cityBounds = {
+          north: city.northLat,
+          south: city.southLat,
+          east: city.eastLng,
+          west: city.westLng
+        };
+      }
+    });
+
+    this.options = {
+      bounds: this.cityBounds,
+      strictBounds: true,
+      types: ['address'],
+      componentRestrictions: { country: 'UA' }
```

Comment:
can we restrict area here  ?

Reply:
We are doing this by using property 'bounds'